bootstrap.py

```diff
@@ -288,11 +301,24 @@ def install_puppet_agent():
     exec_failexit("/sbin/service puppet restart")
 
 
+def install_pe_agent(org_name):
```

Comment:
can't we merge that and install_puppet? just having a flag whether it's puppet or PE that is going to be installed?